pkgs/tools/misc/go-ios/default.nix

```
@@ -2,34 +2,51 @@
 , buildGoModule
 , fetchFromGitHub
 , nix-update-script
+, pkg-config
+, libusb1
 }:
 
 buildGoModule rec {
   pname = "go-ios";
-  version = "1.0.121";
+  version = "1.0.143";
 
   src = fetchFromGitHub {
     owner = "danielpaulus";
     repo = "go-ios";
     rev = "v${version}";
-    sha256 = "sha256-zWaEtfxrJYaROQ05nBQvM5fiIRSG+hCecU+BVnpIuck=";
+    sha256 = "sha256-6RiKyhV5y6lOrhfZezSB2m/l17T3bHYaYRhsMf04wT8=";
   };
 
-  vendorHash = "sha256-0Wi9FCTaOD+kzO5cRjpqbXHqx5UAKSGu+hc9bpj+PWo=";
+  proxyVendor = true;
+  vendorHash = "sha256-GfVHAOlN2tL21ILQYPw/IaYQZccxitjHGQ09unfHcKg=";
 
   excludedPackages = [
     "restapi"
   ];
 
-  checkFlags = [
-    "-tags=fast"
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    libusb1.dev
```

`.dev` is the default
```suggestion
    libusb1
```

---

Fixed in latest push, thanks! :) Appreciate you pointing out where also, didn't expect that.